src/main/java/com/gbtec/GenerationPattern.java

```diff
@@ -0,0 +1,35 @@
+package com.gbtec;
+
+public class GenerationPattern {
+
+    public GenerationPattern() {
```

Comment:
Your have only static methods in this class, so this class will never be instantiated. To prevent the instantiation you can change the visibility of the constructor from public to static. Also remove the unwanted empty line from the constructor.